src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Logs.component.tsx

```diff
@@ -262,6 +262,31 @@ function LogsComponent({ selectedTab, isDeleted, logSearchTerms, setLogSearchTer
 
     let podContainerOptions = getPodContainerOptions(isLogAnalyzer, params, location, logState);
 
+    const getPodGroups = () => {
+        let allGroupPods = [], individualPods = []
```

Comment:
use `const` as we're not doing any reassignment.

Reply:
Done